<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtPrincipal</class>
 <widget class="QMainWindow" name="qtPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QWidget" name="qtPrincipalForm">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>171</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QTimeEdit" name="campo_hora_inicio"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_hora_final">
       <property name="text">
        <string>Hora Final</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTimeEdit" name="campo_hora_final"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_total">
       <property name="text">
        <string>Total Horas</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="campo_total_horas">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_hora_inicio">
       <property name="text">
        <string>Hora Inicio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>611</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLineEdit" name="campo_descricao_tarefa"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_salvar_dados">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_descricao_tarefa">
       <property name="text">
        <string>Descrição da Tarefa</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>202</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id_tarefa">
       <property name="text">
        <string>Tarefa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="campo_data_tarefa">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_data_tarefa">
       <property name="text">
        <string>Data Tarefa</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="campo_tarefa">
       <property name="inputMask">
        <string>000000000; </string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="lista_tarefas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="campo_id_tarefa">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::NoEcho</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_excluir">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuSicronizar">
    <property name="title">
     <string>Sincronizar</string>
    </property>
    <addaction name="actionSincronizar"/>
   </widget>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuSicronizar"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDeletar_Banco">
   <property name="text">
    <string>Deletar Banco</string>
   </property>
  </action>
  <action name="actionSincronizar">
   <property name="text">
    <string>Sincronizar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>campo_tarefa</tabstop>
  <tabstop>campo_data_tarefa</tabstop>
  <tabstop>campo_hora_inicio</tabstop>
  <tabstop>campo_hora_final</tabstop>
  <tabstop>campo_descricao_tarefa</tabstop>
  <tabstop>btn_salvar_dados</tabstop>
  <tabstop>btn_excluir</tabstop>
  <tabstop>campo_total_horas</tabstop>
  <tabstop>lista_tarefas</tabstop>
  <tabstop>campo_id_tarefa</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_salvar_dados</sender>
   <signal>clicked()</signal>
   <receiver>qtPrincipal</receiver>
   <slot>salvar_dados()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>qtPrincipal</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSobre</sender>
   <signal>triggered()</signal>
   <receiver>qtPrincipal</receiver>
   <slot>abrir_sobre()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSincronizar</sender>
   <signal>triggered()</signal>
   <receiver>qtPrincipal</receiver>
   <slot>abrir_sincronizacao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_excluir</sender>
   <signal>clicked()</signal>
   <receiver>qtPrincipal</receiver>
   <slot>excluir_dados()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>salvar_dados()</slot>
  <slot>abrir_sobre()</slot>
  <slot>abrir_sincronizacao()</slot>
  <slot>excluir_dados()</slot>
 </slots>
</ui>
